build.proj

```diff
@@ -1,4 +1,10 @@
-<Project DefaultTargets="Init" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">	
-	<Import Project="Directory.Build.props" />	
-	<Import Project="Directory.Build.targets" />
+<Project Sdk="Microsoft.Build.Traversal">
+  <ItemGroup>
```

Comment:
@weshaggard won't it be easy for now until everything settles down where we have
instead of build.proj, we create
mgmt.proj
<takeYourPick>.proj (for non-mgmt projects)

Reply:
we have that which is `eng/service.proj`